README.md

```diff
@@ -95,39 +95,41 @@ Confirm your PHP installation has curl enabled
 
 This quick example assumes that you have a local riak cluster running on port 8098
 
-    require_once('riak-php-client/src/Basho/Riak/Riak.php');
-    require_once('riak-php-client/src/Basho/Riak/Bucket.php');
-    require_once('riak-php-client/src/Basho/Riak/Exception.php');
-    require_once('riak-php-client/src/Basho/Riak/Link.php');
-    require_once('riak-php-client/src/Basho/Riak/MapReduce.php');
-    require_once('riak-php-client/src/Basho/Riak/Object.php');
-    require_once('riak-php-client/src/Basho/Riak/StringIO.php');
-    require_once('riak-php-client/src/Basho/Riak/Utils.php');
-    require_once('riak-php-client/src/Basho/Riak/Link/Phase.php');
-    require_once('riak-php-client/src/Basho/Riak/MapReduce/Phase.php');
-
-    # Connect to Riak
-    $client = new Basho\Riak\Riak('127.0.0.1', 8098);
-
-    # Choose a bucket name
-    $bucket = $client->bucket('test');
-
-    # Supply a key under which to store your data
-    $person = $bucket->newObject('riak_developer_1', array(
-        'name' => "John Smith",
-        'age' => 28,
-        'company' => "Facebook"
-    ));
-
-    # Save the object to Riak
-    $person->store();
-
-    # Fetch the object
-    $person = $bucket->get('riak_developer_1');
-
-    # Update the object
-    $person->data['company'] = "Google";
-    $person->store();
+```php
+require_once 'riak-php-client/src/Basho/Riak/Riak.php';
```

Comment:
@Bilge that's completely fine, I was simply trying to help there.

Autoloading is completely independent by composer, it's part of of the PHP core 

http://php.net/manual/en/language.oop5.autoload.php
http://php.net/manual/en/function.spl-autoload.php

and it should be taken as good practice and somehow enforced if applicable. 
Mine was simply a suggestion to make your PR even more valuable. 

Same goes for the `require_once`, since I assume you made this PR with the intent of improving the project, I was suggesting to favour `require` over `require_once` because it is not only faster but it also helps you spot duplication errors. If you don't feel comfortable in implementing those changes I will do so once you PR is merged.